sdks/python/apache_beam/internal/metrics/metric.py

```diff
@@ -136,3 +165,64 @@ def log_metrics(self, reset_after_logging=False):
           self._last_logging_millis = current_millis
       finally:
         self._lock.release()
+
+
+class ServiceCallMetric(object):
+  """Metric class which records Service API call metrics.
+
+  This class will capture a request count metric for the specified
+  request_count_urn and base_labels.
+
+  When call() is invoked the status must be provided, which will
+  be converted to a canonical GCP status code, if possible.
+
+  TODO(ajamato): Add Request latency metric.
+  """
+  def __init__(self, request_count_urn, base_labels=None):
+    # type: (str, Optional[Dict[str, str]]) -> None
```

Comment:
fyi - from now on, you can use type hints directly, not as comments (no need to change anything atm)